spec/dummy/app/policies/application_policy.rb

```diff
@@ -34,6 +34,10 @@ def destroy?
     false
   end
 
+  scope_for :default do |relation|
+    relation.all
+  end
```

Comment:
You gotta refer to scopes by name. Maybe we'd want to call it `avo_scope` or something.